app/src/main/java/ca/llamabagel/transpo/ui/map/MapFragment.kt

```diff
@@ -30,8 +53,72 @@ class MapFragment : Fragment() {
 
     override fun onActivityCreated(savedInstanceState: Bundle?) {
         super.onActivityCreated(savedInstanceState)
-        viewModel = ViewModelProviders.of(this).get(MapViewModel::class.java)
-        // TODO: Use the ViewModel
+        viewModel = ViewModelProviders.of(this, requireActivity().injector.mapViewModelFactory()).get(MapViewModel::class.java)
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        val mapView = view.findViewById<MapView>(R.id.mapView)
+        mapView.onCreate(savedInstanceState)
+        mapView.getMapAsync { map ->
+            map.setStyle(Style.TRAFFIC_DAY) {
```

Comment:
Any particular reason for having traffic layer on?

Reply:
Traffic seems like it could be useful information to display when considering transit options.